solidity/contracts/system/TBTCSystem.sol

```diff
@@ -142,13 +143,32 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
             _keepThreshold,
             _keepSize
         );
+        vendingMachine = _vendingMachine;
         setTbtcDepositToken(_tbtcDepositToken);
-        _initialized = true;
+        initialized = true;
         allowNewDeposits = true;
     }
 
-    /// @notice gets whether new deposits are allowed.
-    function getAllowNewDeposits() external view returns (bool) { return allowNewDeposits; }
+    /// @notice returns whether new deposits should be allowed.
```

Comment:
Sentences start with caps, we should probably give more detail on how this is decided, and we should probably set `@return` as well.